submissions/adekemiAwomolo/html/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang ='en'>
+<head>
+	<meta charset="utf-8">
+	<title>E-Commerce Task</title>
+
+</head>
+<body>
+	<div class='outer'>
+		<header>
+<nav>
+	<h3 class='paddy'><a href="#">MyPaddy</a></h3>
+	<p class='paddy'><i>Buying made easy.</i></p>
+</nav>
+</header>
+<aside>
+<br>
+<div class='product'>
+Groceries <span>Fresh foods.</span>
+<span>Vegetables.</span>
+<span>Fruits.</span>
+<span>Cart.</span>
+</div>
+<br>
+
+<br>
+
+</aside>	
+<h2 class='divider'> Products</h2>
+
+<section>
+	<div class='imm'>
+	<img src='asset/it.jpg' height='350' width='690'>
```

Comment:
Remove width and height here @kemysola

Reply:
Good afternoon ma @ToluAkin .I have removed the width and the height on the image as requested.Kindly review ma, thank you.